packages/cardano-graphql-services/test/Http/HttpServer.test.ts

```diff
@@ -26,18 +28,18 @@ describe('HttpServer', () => {
 
   it('Is a runnable module', async () => {
     port = await getRandomPort();
-    httpServer = SomeHttpServer.create({ host: 'localhost', port });
+    httpServer = SomeHttpServer.create({ listen: { host: 'localhost', port } });
     expect(httpServer).toBeInstanceOf(RunnableModule);
   });
 
   beforeEach(async () => {
     port = await getRandomPort();
     apiUrlBase = `http://localhost:${port}`;
-    httpServer = SomeHttpServer.create({ host: 'localhost', port });
   });
 
   describe('initialize', () => {
     it('initializes the express application', async () => {
+      httpServer = SomeHttpServer.create({ listen: { host: 'localhost', port } });
```

Comment:
This has many occurrences. I think wrapping it in a fn would be nice

```suggestion
const createHttpServer = () => SomeHttpServer.create({ listen: { host: 'localhost', port } });
...
      httpServer = createHttpServer();
```

Reply:
The repetition is a bit awkward, but we don't gain much by lifting to a function since `port` is being set in the `before` block, and it would create some disconnect when reading the test. I'll leave it as-is for now, but we can revisit it should the interface get more complicated.

Reply:
Sounds reasonable